extra/modules/fiftyone-devicedetection/src/main/java/org/prebid/server/hooks/modules/fiftyone/devicedetection/core/device/WritableDeviceInfo.java

```diff
@@ -0,0 +1,44 @@
+package org.prebid.server.hooks.modules.fiftyone.devicedetection.core.device;
+
+import java.math.BigDecimal;
+
+/**
+ * A writable counterpart to {@link DeviceInfo}
+ */
+public interface WritableDeviceInfo {
```

Comment:
I believe this interface is only needed to support the current approach, enriching the device in place will make this redundant (as well DeviceInfo)

Reply:
this was removed